server/option.go

```diff
@@ -145,9 +145,9 @@ func WithLimit(lim *limit.Option) Option {
 	}}
 }
 
-// WithConcurrencyLimiter sets the limiter of concurrent connections.
+// WithConnectionLimiter sets the limiter of connections.
 // If both WithLimit and WithConcurrencyLimiter are called, only the latter will take effect.
```

Comment:
WithLimit and WithConcurrencyLimiter => WithLimit and WithConnectionLimiter

Reply:
done